src/sentry/static/sentry/app/stores/streamTagStore.jsx

```diff
@@ -4,8 +4,13 @@ import _ from 'lodash';
 import StreamTagActions from '../actions/streamTagActions';
 import MemberListStore from './memberListStore';
 
+const getUsername = user => {
+  const useUsername = !user.isManaged && user.username;
```

Comment:
may read nicer to pull ismanaged, username, and email all off user with object destructuring

also, this is pedantic, but we've agreed to use let inside function bodies for variables that aren't reassigned. :/ I dont mind merging this as-is